### DISCUSSION TITLE
How the relay should be built

### DISCUSSION THREAD

**Discussion by FlameArrow57** [stonepillars/goon-flock#204] [Ideas]
Decision discussion about how the relay should be built. There were two ideas about how it should be built.

The first one would be that the relay is simply built upon requiring a certain amount of compute.

The second one was that the relay requires a certain number of arrays to be built.

**Comment by FlameArrow57** [comment]
I think the first option is a good prerequisite but feels a bit bland/lacking just having a compute requirement for its building. I think requiring building of relays would add a goal for the flock to work towards along the way towards building the relay.

Arrays would also help with how powerful the flock is since the flock could be more spread out. There might be an issue with how much of the station needs to be converted for this to happen, since if you going on a continual path from your starting point to build each of the arrays, shuttle might've been called already if allowed or a lot of people dead, with much of the station converted. Could balance this though, as Flock could try to establish beachheads instead at certain points or could make the distance requirement not super far.

Also, there was discussion about arrays influencing how the flock talks, ex. clarity of radio comms, as well as allowing new abilities to be added, and could also unlock new structures based upon building arrays. I think that would be a good design point too to work off of, much more interesting than just needing to meet a certain point requirement.

**Reply by stonepillars** [reply]
We'll never really know how well the flock can spread with people actively trying to kill it without playtesting to find out. It's possible that the crew can provide enough resistance to keep the flock from even unlocking the relay, thus making the arrays a pretty hard hurdle to cross

**Reply by amylizzle** [reply]
It's already gonna be hard to defend the relay from a concerted attack from crew. See: nukies.
